.github/workflows/default.yml

```diff
@@ -28,18 +28,28 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
-      - uses: actions-rs/toolchain@v1
+      - name: Install nightly `rust-fmt` 
+        uses: actions-rs/toolchain@v1
         with:
             # NOTE: Until https://github.com/actions-rs/toolchain/pull/209 is merged,
             # this should be synced with rust-toolchain.toml file.
             # After merge, this section should be removed.
             toolchain: nightly-2021-12-05
-            components: rustfmt, clippy, llvm-tools-preview
+            components: rustfmt
+            profile: minimal
+
+      - name: Install Rust toolchain 1.64 
```

Comment:
Maybe it would beneficial if we have short comment for reasoning behind this change (hopefully someone will check in the future and improve it 🤷 ). Btw, PR from note above is still not merged :D